chromium_src/chrome/browser/ui/views/frame/browser_view_layout.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/tabs/brave_vertical_tab_utils.h"
+
+#define BRAVE_SKIP_IF_VERTICAL_TAB \
+  if (ShouldShowVerticalTabs())    \
```

Comment:
Maybe we can add `BraveBrowserViewLayout` and handle this via `BraveBrowserViewLayout::LayoutTabStripRegion()` without adding new patch?